### ISSUE TITLE
Documentation incorrect re: `pr` attribute

### ISSUE THREAD

**Issue by Lykathia** [google-github-actions/release-please-action#438]
### TL;DR

The documentation says this returns the PR #, however it returns the [PR Object](https://github.com/googleapis/release-please/blob/7584fecd34794ed146117c80a7f37f8e64f9d0bf/src/pull-request.ts#L15).

I think this is probably better. I'd like to be able to get the version, for use in other actions (tho that doesn't seem available atm). However, the tests are hiding this via mocks, and the documentation doesn't line up.

### Action YAML

```yaml
  release-please-pr:
    runs-on: ubuntu-latest
    permissions:
      contents: write
      pull-requests: write
    outputs:
      pr-created: ${{ steps.release.outputs.pr }}
    steps:
      - name: Generate Release Pull Request
        uses: google-github-actions/release-please-action@v3.1.1
        id: release
        with:
          release-type: simple
          command: release-pr
```

### Log output

```text
::set-output name=pr::{"headBranchName":"release-please--branches--master","baseBranchName":"master","number":1,"title":"chore(master): release 1.0.0","body":":robot: I have created a release *beep* *boop*\n---\n\n\n## 1.0.0 (2022-02-17)\n\n\n### Features\n\n* release test ([62736ec](https://github.com/redacted/validate-adv-release/commit/62736ec92e87a97614a8637d5b912ff3e500129c))\n\n\n### Bug Fixes\n\n* try again ([5f2812d](https://github.com/redacted/validate-adv-release/commit/5f2812db684a34b8f4913ffb1a282af14bca9ef6))\n\n---\nThis PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).","files":[],"labels":["autorelease: pending"]}
##[debug]steps.release.outputs.pr='{"headBranchName":"release-please--branches--master","baseBranchName":"master","number":1,"title":"chore(master): release 1.0.0","body":":robot: I have created a release *beep* *boop*\n---\n\n\n## 1.0.0 (2022-02-17)\n\n\n### Features\n\n* release test ([62736ec](https://github.com/redacted/validate-adv-release/commit/62736ec92e87a97614a8637d5b912ff3e500129c))\n\n\n### Bug Fixes\n\n* try again ([5f2812d](https://github.com/redacted/validate-adv-release/commit/5f2812db684a34b8f4913ffb1a282af14bca9ef6))\n\n---\nThis PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).","files":[],"labels":["autorelease: pending"]}'
```

### Additional information

I would just submit a PR to correct this, however if it is a regression I am not sure which direction to take it in. Personally - I would much rather have the object and data available similar to how the `github-release` command works - but that doesn't seem to be written intent.

**Comment by live627** [comment]
it was just  forgotten because v2 really only returns a number instead of an object.
